chia/wallet/transaction_record.py

```diff
@@ -109,3 +110,17 @@ def to_json_dict_convenience(self, config: Dict) -> Dict:
             if memo is not None
         }
         return formatted
+
+    @property
+    def is_spendable(self) -> bool:
```

Comment:
it makes me nervous to have properties like this. The risk is that it looks cheap (just querying a state) whereas in fact it's not. I suppose the only way for this function to be expensive is if `self.sent_to` is a very long list.

I'm not familiar with this code, but this name seems odd to me. it looks like it's answering the question "do we still think there is a chance for this transaction to land in a block?". the name `is_spendable()` implies a few other things in my mind. e.g. (1) is it valid, (2) is it unspent.